src/lib/pages/frontsite/Literature/LiteratureSearch/LiteratureSearch.js

```diff
@@ -53,72 +54,82 @@ class LiteratureSearch extends Component {
     const defaultSortingOnEmptyQueryString = setReactSearchKitDefaultSortingOnEmptyQueryString(
       this.modelName
     );
+    const { extraMap } = this.props;
+
     const urlHandler = setReactSearchKitUrlHandler(this.modelName);
     return (
-      <OverridableContext.Provider
-        value={{
-          ...SearchControlsOverridesMap,
-          ...LiteratureSearchOverridesMap,
-          ...SearchBarOverridesMap,
-        }}
-      >
-        <ReactSearchKit
-          searchApi={this.searchApi}
-          history={history}
-          urlHandlerApi={urlHandler}
-          overridableId="LiteratureSearchOverridable"
-          initialQueryState={initialState}
-          defaultSortingOnEmptyQueryString={defaultSortingOnEmptyQueryString}
+      <Overridable id="LiteratureSearch.extraMap">
```

Comment:
why `extraMap`?
here we make overridable the entire search app, is this what we want?
Why not using the `LiteratureSearchOverridesMap` prop?